tools/ARIAtools/mask_util.py

```diff
@@ -255,12 +255,12 @@ def arr2ds(ds_orig, arr, noData=0):
 
 
 class NLCDMasker(object):
-    def __init__(self, path_aria, lc=[11, 12, 90, 95]):
+    def __init__(self, path_aria, lc=None, dem_name='glo_90'):
```

Comment:
Changed the default argument because it will lead to horribly confusing errors.

https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument